exporters/jaeger/src/test/resources/tls-test.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Let's use `SelfSignedCertificateExtension` instead of checking dummy certs into code. Example available [here](https://github.com/open-telemetry/opentelemetry-java/blob/main/integration-tests/otlp/src/main/java/io/opentelemetry/integrationtest/OtlpExporterIntegrationTest.java#L118-L122).

Reply:
Good call, it felt bad adding those, and I had even touched some uses of that extension elsewhere, so I should have known. 👍🏻